src/test/resources/copyGraph.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+wget https://github.com/amcp/dynamodb-import-export-tool/archive/createTable.zip
+unzip -q createTable.zip
+rm createTable.zip
+pushd dynamodb-import-export-tool-createTable
+mvn install
+pushd target
+
+export ORIGINAL_TABLE_ENDPOINT=$1
+export ORIGINAL_TABLE_REGION=$2
+export ORIGINAL_TABLE_PREFIX=$3
```

Comment:
take in old table name and new table name as parameters and do not iterate over all graph stores in this file. Need flexibility to support the upgrade path from Titan 1.0.0 to JanusGraph 0.1.0 given the titan_ids/janusgraph_ids table name difference.